slider/replay.py

```diff
@@ -134,19 +137,24 @@ def _consume_actions(buffer):
     decompressed_data = lzma.decompress(compressed_data)
 
     out = []
+    abs_offset = 0
     for raw_action in decompressed_data.split(b','):
         if not raw_action:
             continue
-        offset, x, y, raw_action_mask = raw_action.split(b'|')
+        raw_offset, x, y, raw_action_mask = raw_action.split(b'|')
         action_mask = ActionBitMask.unpack(int(raw_action_mask))
+        rel_offset = int(raw_offset)
+        abs_offset += rel_offset
         out.append(Action(
-            datetime.timedelta(milliseconds=int(offset)),
+            datetime.timedelta(milliseconds=rel_offset),
```

Comment:
a timedelta64 can be converted into seconds, or ms by viewing it as an int, for example:
```python
>>> delta = np.timedelta64(100, 'ms')
>>> delta.view('int64')
100
```